nanoFramework.Json/JsonSerializer.cs

```diff
@@ -125,9 +125,56 @@ private static bool ShouldSerializeMethod(MethodInfo method)
                 return false;
             }
 
+            //Ignore property getters with at least one parameter (this[index] operator overloads)
+            ParameterInfo[] parameters = method.GetParameters();
+            if (parameters.Length > 0)
+            {
+                return false;
+            }
+            parameters = null;
+
+            // Ignore properties listed in [JsonIgnore()] attribute
+            if (ShouldIgnorePropertyFromClassAttribute(method))
+                return false;
+
+            // per property [JsonIgnore()] attribute - not working due to method.GetCustomAttributes returning empty
+            //var attributes = method.GetCustomAttributes(false);
+            //foreach (var attributeInfo in attributes)
+            //{
+            //    if(typeof(JsonIgnoreAttribute).IsInstanceOfType(attributeInfo))
+            //    {
+            //        return false;
+            //    }
+            //}
+
             return true;
         }
 
+        /// <summary>
+        /// split out method to check for ignore attribute
```

Comment:
For all the comments, please start with uppercase and please finished by a dot.
Please apply this to all the intellisense you're bringing in :-)
And yes, there are quite a few to fix elswhere